packages/web-twig/src/Resources/components/CheckboxField/stories/CheckboxFieldItem.twig

```diff
@@ -1,13 +1,40 @@
 <CheckboxField
-    id="checkboxfield18"
+    id="checkboxfieldItemDefault"
     isItem
-    label="Checkbox Item unselected"
-    name="checkboxfield18"
+    label="Checkbox Label"
+    name="checkboxfieldItemDefault"
 />
+
 <CheckboxField
-    id="checkboxfield19"
+    id="checkboxfieldItemDefault"
+    isChecked
+    isItem
+    label="Checkbox Label"
+    name="checkboxfieldItemDefault"
+/>
+
+<CheckboxField
+    helperText="Helper text"
+    id="checkboxfieldItemHelperText"
+    isItem
+    label="Checkbox Label"
+    name="checkboxfieldItemHelperText"
+/>
+
+<CheckboxField
+    id="checkboxfieldItemDisabled"
+    isDisabled
+    isItem
+    label="Checkbox Label"
+    name="checkboxfieldItemDisabled"
+/>
+
+<CheckboxField
+    helperText="Helper text"
+    id="checkboxfieldItemDisabledHelperText"
     isDisabled
     isItem
-    label="Checkbox Item unselected disabled"
-    name="checkboxfield19"
+    isRequired
+    label="Checkbox Label"
+    name="checkboxfieldItemDisabledHelperText"
 />
```

Comment:
- ⚠️ label text should be bold
- checkbox should be checked

Web:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/ce3e6be6-9cf2-4ebf-bcf9-267144c35f16)

Twig:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/166c430f-ba24-472d-86ef-5d022c602b37)

Reply:
Fixed.